src/main/java/world/bentobox/bentobox/api/commands/island/IslandGoCommand.java

```diff
@@ -31,6 +32,11 @@ public boolean execute(User user, String label, List<String> args) {
             user.sendMessage("general.errors.no-island");
             return false;
         }
+        if ((getIWM().inWorld(user.getWorld()) && !Flags.PREVENT_TELEPORT_WHEN_FALLING.isSetForWorld(user.getWorld()))
```

Comment:
`isSetForWorld` will return the current "value" for this flag - at least, that's what I understood from others WORLD_SETTING flags using the same method. And... Oh. Uh... Did I...?

Wait, we want the flag to prevent teleportation when it is **true**, right?

Crap.